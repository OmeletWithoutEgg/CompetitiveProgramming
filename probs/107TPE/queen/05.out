-7383487.07 43586568.90
