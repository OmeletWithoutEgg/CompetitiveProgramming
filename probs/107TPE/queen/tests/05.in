854
-58715706 -29044798
92648059 -2983545
-108117001 -56922772
-61762594 -5080071
-62023073 -4328203
42114358 67284493
78313006 69402531
-67071991 -55905254
-73050205 -62880490
60497045 -14481860
106693923 25536911
-99233212 -65576758
-117116395 -20150361
38748917 65366470
-107757117 7478344
-71545867 -61459504
43287449 67865300
98560515 54101818
-59061328 -17119231
85323364 -8370599
106822477 28642771
-64251459 998098
-102922960 12854280
-115932562 -38211758
-89034742 -68931581
68064925 -14200559
91249935 61899214
25904585 4675401
-89832315 18924372
98379244 3349906
-74070962 13794136
99442571 4851524
-76762742 15700508
106559287 24118160
-73968627 -63657873
-71835501 11801740
-112053258 111358
-99970300 -65098520
28818548 56816827
-69051594 8695407
-86170300 -68881938
-78467214 16661123
80076294 -11003954
26291501 4119050
79428744 -11271196
96461679 56755173
-84157047 -68594930
27970005 55798620
-116342047 -36081536
-91146888 18752650
-89912018 -68862863
-58677201 -28367247
-74875813 -64363516
-58812096 -19568396
-98169246 16258789
-111217686 -51771842
19456008 19443441
18418501 27752010
106619279 24689600
-115482841 -9762905
-97956490 16375923
46614818 69289664
64715597 -14482549
78572221 69304993
-59148896 -33518808
18529393 32109503
-65515541 -53353438
21096111 13943225
82463488 67611361
-111206552 -51793550
23122430 9349381
106728108 26008730
-60679052 -8673582
93702448 59691798
-59716503 -37044535
25275277 52076376
-116998915 -31188092
-86496835 18962422
106581557 33376570
57446873 -14234323
-63068899 -1616781
-58590665 -24616708
-61111791 -42812793
-87548945 -68958168
-85762938 -68840766
19365731 19851563
-60072648 -11167826
-91500157 18691237
-109224936 5318853
103592064 45128440
100651634 50956402
-73213765 13077374
-84736997 -68699155
-109179409 5390537
-105762220 9980145
-84873430 -68721128
-100495008 14789516
53365157 71270081
61734019 -14523020
69659088 71675539
105580284 18635924
-110161442 -53715162
106316448 35397203
96848232 1400961
99767058 5338065
65479290 72138728
28808045 893747
64028981 72207722
-59505961 -14067970
33526468 -3819498
88489411 -6324670
75883658 -12531233
102966653 11129375
-113634272 -46242356
-63065558 -48321592
22927408 9734418
-102175481 -63443503
68985815 71776971
-106529604 -59017006
-72794480 12705869
-61127461 -7080733
82152925 -10064649
-67323599 -56281944
-91988695 -68541931
-65301410 3022611
18929136 22184945
22784445 47675440
106821611 29190283
-65235156 2901834
80102861 68691084
-74370528 14031544
-78947100 16899503
-116910109 -32058306
69852611 71644452
104379930 43070405
-85085933 18807065
55514192 71666942
29193297 451231
105334389 40011532
-59690953 -36909542
21026011 14132588
92622397 -3006841
45608348 68892263
-58744191 -20465873
-83914792 -68546186
83411867 67129571
98822274 3959217
-58649566 -22165130
-104064722 -61731188
29157610 491720
20850306 43078598
40502843 -8716189
94421576 -1279153
60561570 72183198
18905708 22334534
-104480448 -61313975
37519364 -6865673
-89977511 18909783
22226836 46494568
101337663 49778195
-72775131 -62634748
105092697 16831449
-99898338 -65146800
-103400495 12420236
39116724 -7897064
57961746 71989127
102753977 47028137
23084193 48274227
-64121273 -50675620
22236946 46516832
28661435 56632996
-102304696 13389118
-101905977 13718348
-114005556 -4744072
-72978637 12870774
-117199990 -21467715
103462160 12258533
-58897531 -31327286
-108031912 -57042976
40673749 -8812551
19584451 38805421
63487877 -14523126
30287253 -742998
-116607444 -34427216
-93677704 18167540
66312316 72077872
78867550 -11493274
22439877 46956779
-107057226 8407962
-103721360 -62064350
106598413 33216080
98975405 4175102
105249035 17376897
93148836 -2521456
80154544 68669201
73538719 -13186137
18562703 32517487
-59067574 -32878546
-109734012 -54441034
105959308 20316253
84141447 66739215
-60666165 -41224287
77160738 -12115842
-69578634 -59290407
-117115053 -20132163
-111833835 -50524936
-60530539 -9242964
-116162125 -12878691
-116496245 -35155029
18845455 22736436
-103504591 -62269491
95790298 177359
36752403 -6334920
-117293153 -24817261
-96639549 17039907
105309055 17593875
-82715108 -68259136
45264317 -11051561
-66852378 5622514
-110915530 2404838
32643045 60739711
-58722741 -29156663
77811818 69585954
-95372235 -67529134
90199335 62748712
-91826420 -68575097
101456520 8133313
20879663 14537055
-83490201 -68453311
-67583628 6715456
-82788711 -68278951
105450752 18124803
-112186552 -49767477
90999673 62106441
-79510069 -67108155
-88123870 19014889
73359764 70928879
87217905 64890840
-90710599 18819544
80863969 -10662673
19470440 38318443
32681254 -3075897
53828378 -13666432
-116294530 -13595462
78743266 69239625
-66232490 -54579080
-70864926 10804725
-71709346 -61623311
-102155899 13513397
-61852440 -45129655
-82158527 -68099867
-106789233 8748498
98020301 2872052
47496076 69614805
22423213 10777209
-112092182 27115
96144701 57121235
-65432296 3258667
-99312266 -65527177
46205053 69131265
-78685301 16771161
-111744215 765750
-117235052 -27743986
55451374 71656868
25140458 5831977
18422142 30079465
-116582177 -15348690
-115516668 -9901190
78111178 69477211
-113208745 -2588306
106065837 36843401
-63088228 -1570614
-77369118 -66009422
18443905 27014720
-69632933 -59355507
38858590 65435259
-86260216 18943512
-110539469 -53045378
-74280856 -63907538
-58986159 -17767628
-67904269 -57117563
36725705 64014359
93437509 59946786
-95478062 17540950
66728701 72041623
-66632678 5278679
-115067903 -8171369
19655388 39096128
-93622826 18183848
-117013219 -31035978
-73035912 -62867870
-83046194 18399627
-67998800 7302965
-91088370 18762196
-106262175 9394700
106752111 31304452
89592269 -5517567
56300309 71785334
-88377756 -68957244
-106094466 9594002
102558555 10260361
71977419 71245732
-81815038 18046847
-116264115 -13426571
48841425 70071261
81638099 -10309503
91818615 -3718191
-70523115 -60379059
49519170 -12585028
65633306 72128646
-105549721 10221201
23368852 48822252
-59684857 -13069282
83945254 -9147557
-111855901 -50478530
-98794823 15898019
-114624202 -6644916
22914354 9760535
23977896 49934604
-75367879 -64721577
22654542 47408496
94546303 58850194
56888235 71864624
88860573 63757221
102894809 46725677
-63429283 -49170193
29960551 -395586
27323640 54978628
18990629 35890585
35197205 62880625
-113049234 -47757616
46776066 -11652383
-58598164 -26029418
-76520896 15549321
106793943 30421135
26038402 4480879
20907236 14459873
-59603205 -36432843
106712663 31911842
-59400349 -35240034
-70683215 -60554716
-59437609 -35469953
21505817 12887267
-59430722 -35427859
-76515525 -65492285
-100739484 -64560449
-61867728 -4772326
-102206726 13471136
35578466 63173060
-87573564 -68958646
18415229 27887272
-117286735 -25906140
68296042 71870122
-109044701 -55547128
-94500844 -67849633
97584292 55388894
-67370546 -56351227
93995831 59404372
80144658 -10975043
-63237862 -1217436
32309646 60434883
99707358 52450875
22955628 48020243
67811960 -14230675
-65031378 -52471150
71474790 -13651277
-117142240 -20515495
73822921 -13114028
-116207531 -13119236
89501870 -5585714
-116234297 -13263533
51315594 70789435
24826153 6332184
45337684 68780550
-107251972 -58101613
28842895 853220
97346329 2010255
58031408 71996305
-98503311 -66015558
-59761800 -37279932
-79488877 -67098617
-91163696 -68696240
-81550821 -67906790
43267915 67855977
-67874409 7129334
-70806421 10741843
-106608546 8973487
-87129878 18998916
30014455 -453432
26794863 54276399
54321181 71459975
-83158584 18427782
-100871703 14517465
106784149 27031421
-112290703 -409194
92437880 60871603
-90631380 -68776998
32560657 60664967
106799412 27432944
-109929469 4166730
23327112 48743084
60432071 -14478757
35301019 62960908
-87634210 19013326
-98297634 16186747
94470167 58928029
-105909101 -59757210
-109244353 5288183
-88867090 18993861
-104225342 -61571815
90663478 62379976
100917457 50509959
-115313153 -9089820
-70051306 -59846662
38936819 -7785705
-104846179 10987645
67849189 -14226334
106694451 32154667
-59014283 -32428136
96719802 56450905
18802409 34657410
74239704 -13004695
-87082508 -68943260
-105283641 -60463066
96265156 56983094
95403182 -248272
-85005852 18795167
-70293775 -60123052
35424528 -5357431
-60929267 -42183031
-114633520 -43270948
19242909 37258295
105991440 37223401
-96269686 -67154265
61533214 72217000
87098375 -7270784
19426467 19574922
18706850 23783147
96863213 1418962
35511211 -5423603
-59100358 -33142921
71657157 71312622
57841700 71976500
-94077332 -67990225
62865799 72229252
47675968 69678607
19335801 19991218
-58590010 -24771453
-66918401 -55670763
18435926 27210135
-82597261 18280440
-67827904 7063899
-114976758 -7844592
91244133 -4205704
106161421 36325845
-117267647 -26813252
-98724837 -65885971
-115718425 -39188737
-95289131 17615167
-100016603 -65067272
-93913991 -68041826
20636344 15239455
-110746846 2719786
69522000 71697037
66907462 -14326516
-66859234 5633125
105820623 38034412
-64219543 932583
19049083 36230821
-117273766 -23367033
-58688786 -28584590
-104275479 11575235
106526142 23830570
-101576675 -63927616
-61758493 -5092196
25200062 51959544
-74661679 -64202348
19699141 39271157
-67158721 6089769
-116668976 -33997969
77174952 -12110979
-76799227 -65669352
26797945 3417770
80244760 -10932470
-85873552 18906425
-58702030 -21128412
18734281 34139092
81680621 67987934
97694318 55248709
-117248305 -27408964
-69518596 -59218058
-78969518 16910304
-93009957 -68301309
58352199 -14329599
102129659 9397723
-109454689 4952107
-96852854 -66885749
61560532 -14519298
-80968586 17756418
-64316927 1131698
39626778 -8206392
-102691194 13058299
18450749 26862389
-65960763 -54125282
-76881194 -65719542
-113872746 -4364279
-74035722 13765771
70278039 71573066
-89927503 -68861295
106660018 25127875
-116756204 -16596236
-117087426 -30174561
68546452 -14139212
92662762 60668531
-108462401 6478445
-61197164 -43098429
-67695629 6876213
84003705 66814253
-77199951 15964265
96984529 1565463
-59066279 -17078450
-99544380 15432875
-88084092 -68961584
97943872 54926300
-68849085 -58384227
101306868 49832964
-105956142 -59702519
57665436 -14259020
21339399 44392071
-89951671 18912459
81052933 68276456
-76384886 15462620
99771589 52353389
-113992259 -45240726
91333251 -4131182
-116890387 -17709263
37166451 -6624419
61473205 -14517172
19570974 38749171
-110296023 -53479808
-58836491 -19280863
25306237 52124228
-86738719 -68925153
-109963113 4109611
-102020666 -63571248
-66080015 -54325998
106069561 36823885
-59374700 -35078543
-104535112 11311598
29026947 640834
91151048 61981468
-66552367 -55097479
-115850227 -38596589
-115584651 -39762826
75986439 70197711
-77153335 -65883089
105245923 17365765
50623276 -12905599
25053950 5968147
63961712 72209786
83733527 66959619
54941896 71571787
-78465241 16660115
94002586 -1699341
24901511 51487485
78183357 69450630
-66591357 -55159538
-75730842 -64974929
32479494 -2892613
-61956948 -45430618
-114043048 -4852939
30735680 58906098
-114679351 -43120037
-112271559 -49579739
-109790869 -54346319
-66732185 -55381698
26944841 3219754
31573148 -2040216
92980137 -2678734
-74058861 -63730771
81285167 -10472730
79609830 68896007
-103913992 -61878695
95093323 -580991
-80063494 17401678
-61428148 -43845243
18972773 21915118
59300945 -14409647
-117234854 -27748683
30495829 -960877
-115538541 -39954987
-93615139 -68132484
-96984541 16875984
-61953940 -45422033
-101677931 13901269
-82547566 18266583
-103280999 -62476966
100201549 6012718
56229488 71775250
-115591100 -39735726
22966394 48041673
106726857 25990087
-103133412 -62611653
-108418337 -56489247
-92114189 -68515334
31141542 59315146
106440059 34546438
-59590444 -36361726
18444761 30703364
-65185892 2811457
-113731710 -45975820
77396607 69732807
22218063 11223128
84022225 -9105852
106493834 34132879
28012692 1846977
50553239 -12886172
18544677 32302731
-79855852 17313739
-90739360 18815438
-117153228 -29266124
101149150 7587765
-62132608 -4022259
82047762 -10115323
-81054868 17787836
61214004 -14509862
20155075 16762115
-100756734 14601548
101347009 7936813
-63354029 -948272
34034105 -4248091
-58782216 -19943285
75489853 -12650854
19193154 20691029
-75051855 14547205
-116655641 -15853580
-102721893 13031520
88650177 63908602
32726206 60814770
57120721 71893805
-72425411 -62313415
-61477723 -5945480
97679849 55267211
-102654792 -63036321
30773265 -1246084
-75244174 -64633156
83611240 67024627
34797346 -4868393
-90182743 18887313
106318517 35383889
105701867 19141643
60748154 -14492969
-91699664 18653677
-77380640 16069721
-111156561 -51890666
52860962 71161215
-96541162 17085373
22470542 47022144
41040132 -9015798
-110723044 -52710117
19550765 19034165
98630890 3693268
60679439 72188418
-64931971 2337724
101316908 7883221
-93213666 -68246677
55088681 -13898567
39330087 -8027602
38328092 65098346
-66823455 -55524038
102312575 47938630
20267829 41311276
-97873921 -66366997
-83789782 -68519828
-78517309 -66633017
-113653265 -3755621
35624495 -5509572
-107625954 -57603508
-102896887 -62823810
63579076 -14521264
87925731 -6717404
-109126011 -55420568
-112853516 -1711227
38478357 -7496503
93152051 -2518443
83233672 -9523931
54058050 -13711507
21033481 43586071
-87801192 19015235
22383826 46836560
42836564 67647133
-58608456 -23400917
-73838265 13605110
-97477089 -66576063
-115361258 -40669081
-82278733 18189314
42434602 67447323
38922046 -7776507
-60699023 -8598758
-72974462 -62813431
-113606368 -3628475
-97014606 16861366
-76740598 15686823
106709730 31952242
-106293334 -59303706
30335809 58492338
48208657 -12164150
-103028841 12759643
40940374 -8960905
-116858539 -17429725
79135016 -11388525
59460591 -14421134
24985290 6077027
-82166751 -68102342
-58629584 -27265544
36704433 -6300922
18511945 25824571
-91089600 18761997
63933069 72210634
51344162 70796835
-82864051 18352569
55165111 71609803
-59092264 -33078477
-112684456 -48636936
85979887 65675589
103180373 11605645
100415871 51342416
80180649 -10959770
106394354 22823046
-58705229 -28872213
-58590579 -25313416
96963863 56157934
41133271 66765098
18417117 29891263
18423508 30125847
-94138203 -67970622
-100741765 -64558793
18498384 26021391
-59575554 -36278144
32104696 60244335
46562806 -11571466
30350612 58507851
64502584 -14491993
-113311249 -47096127
-66740268 -55394356
19957431 17451842
31179158 59352516
-60234079 -39490023
26998188 3148419
34683364 62475893
-71709602 -61623566
106720391 25895602
63862359 -14514301
66448950 72066406
-88271374 -68959325
-60523591 -40676183
-59054876 -32773663
101637704 8463825
-96643724 17037965
66381609 -14373761
-60104299 -38921227
76552978 70017155
-114555548 -43524291
104826812 15960888
-70137829 -59945973
-83340215 18471863
-75418218 -64757255
-78326868 16588797
-115057697 -8134410
-111296835 1670363
-72465293 12404205
20953602 43367227
72444606 -13445430
20396236 41723929
22215359 46469252
35322540 62977489
-116498714 -35139460
-89402730 18961195
30118905 -564927
39487281 65821071
22597023 47288781
-103794046 12048306
99181447 4470059
-58598302 -26038003
25145504 51874279
33376358 61388568
-64781350 2050260
-64118961 724443
-81022700 -67722536
64440165 -14494569
23917231 49827214
36733702 -6321678
104381095 43067109
83732489 66960173
-60965865 -7634880
-117060934 -30497779
96025943 442058
-59113944 -16696446
-59916874 -11895255
97918011 54960003
-106315719 9330438
-92366841 18512906
18470896 26470019
104609932 15295829
96228042 672567
62821151 -14531131
-67330892 -56292727
-114588017 -43419308
-108835150 -55868689
-75357876 -64714467
32246798 60376710
19664793 18564330
-109508226 4865429
-58857357 -30900203
18995867 35921757
-84402165 18694757
49885470 -12694787
-64094289 -50619359
-68355560 -57736331
-117120511 -20206621
57954671 71988392
-58930839 -18286046
-108249980 6786615
-58951464 -18088086
93889399 -1810917
101214560 7702321
65324712 -14449965
90400782 62589924
-75919428 15156657
-104704968 11135812
-97346899 16696209
21001894 43499973
47956047 69776230
36980608 64193700
-58690816 -28621342
65198713 72155733
32009224 -2456386
30415180 -876988
105422937 39679960
106819775 29371360
-60060292 -11223975
-89819352 18925621
-62874613 -47858314
-114985743 -42069884
39725417 65963485
87314024 -7129132
-64208588 910038
-59814055 -37545557
49398082 -12547994
33756233 61713463
89456118 -5620066
24446012 6958234
106333818 35284584
90248756 62709933
