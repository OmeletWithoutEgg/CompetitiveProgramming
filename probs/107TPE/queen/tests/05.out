-16875183.10 2899254.46
